_includes/headertop.html

```diff
@@ -13,9 +15,9 @@
     <meta name="twitter:site" content="@scala_lang"/>
     <meta name="twitter:creator" content="@scala_lang"/>
     {% if page.url %}
-    <meta property="og:url" content="{{ site.baseurl }}{{ page.url }}"/>
+    <meta property="og:url" content="{{ site.url }}{{ page.url }}"/>
     {% endif %}    
-    <meta property="og:image" content="{{ site.baseurl }}/resources/img/scala-spiral-3d-2-toned-down.png"/>
+    <meta property="og:image" content="{{ site.url }}/resources/img/scala-spiral-3d-2-toned-down.png"/>
```

Comment:
Does this work?  I'm not seeing the full URL when I build the site locally.

It seems like the issue is that `baseurl` is empty in the `_config.yml`:

https://github.com/scala/scala-lang/blob/af99785/_config.yml#L5

Changing it to www.scala-lang.org has other issues I predict.

Reply:
Seems I have to set the `site.url` according to https://jekyllrb.com/docs/variables/.
In development it's set automatically: https://jekyllrb.com/news/#3-siteurl-is-set-by-the-development-server.

`baseurl` is something different, this page has a good explanation: https://byparker.com/blog/2014/clearing-up-confusion-around-baseurl/

Reply:
Cool, this looks promising.  It should  fix the bug on deployment, but still does the right thing for local development.

Reply:
Doesn't look like this change will disrupt link checking by htmlproofer, so there's validity in the Travis test passing.  Doesn't seem it's getting bypassed.